File: python_classwork/NumberPredictor.java
import java.util.Scanner;
public class NumberPredictor{
    public static void main(String[] args){
    Scanner input = new Scanner(System.in);
    int fixedValue = 50;
    System.out.println("Enter a number: ");
    int userInput = input.nextInt();
    while (userInput != fixedValue){
    if (userInput > fixedValue){
    System.out.println("Too high");
}
    else{
    System.out.println("Too low");
}
    System.out.println("Enter a number: ");
    userInput = input.nextInt();
    if (userInput == fixedValue){
    System.out.printf("%d is the correct number %n", userInput);
}
}
}
}

File: python_classwork/number_predictor.py
user_input = int(input("Enter a value: "))

fixed_value = 50

while user_input != fixed_value:
    if user_input > fixed_value:
        print("Too high")
    else:
        print("Too low")
    user_input = int(input("Enter a value: "))

if user_input == fixed_value:
        print(user_input, "is the correct value")
